### PR TITLE
[#81860340] Enable explicit AVS for ZIP code

### PR DESCRIPTION
**Pull request by dlinder**
Story: https://www.pivotaltracker.com/story/show/81860340

### CODE REVIEW

**Comment by dlinder** [comment]
Depends on styleseat/styleseat#886

File: satchless/contrib/payment/paymentsgateway_provider/__init__.py [hunk 1]
```diff
@@ -51,6 +52,9 @@ def pg_pay(variant, transaction_type, amount=None, first_name=None,
         kwargs['ecom_billto_postal_name_first'] = first_name
     if last_name:
         kwargs['ecom_billto_postal_name_last'] = last_name
+    if zipcode:
+        kwargs['ecom_billto_postal_postalcode'] = zipcode
+        kwargs['pg_avs_method'] = "10000"  # Test but don't fail on zipcode match
```
Comment (elijahr) [inline satchless/contrib/payment/paymentsgateway_provider/__init__.py]: "10000" should be a constant

**Comment by elijahr** [comment]
Just one comment.  Nice!

File: satchless/contrib/payment/paymentsgateway_provider/__init__.py (hunk 1 above)
Comment (dlinder) [inline satchless/contrib/payment/paymentsgateway_provider/__init__.py]: 4c8a2833aad129cf046f1745fea0977e31b08ce9

**Comment by elijahr** [comment]
:zap: :musical_score: :zap:

**Pull request merged**